app/webpacker/styles/application.scss

```diff
@@ -15,6 +15,7 @@ $govuk-image-url-function: frontend-image-url;
 @import 'task-list';
 @import 'tab-navigation';
 @import 'accessible-autocomplete';
+@import "questions";
```

Comment:
Can we call this `input` and rename `_questions.scss` to `_input.scss` – best to keep file names the same as the namespace they contain.